### ISSUE TITLE
Error: Read must be called first

### ISSUE THREAD

**Issue by mohsad06** [mysql-net/MySqlConnector#379]
When I run the sample piece of code below:

```
using (var reader = await ExecuteGetListReader(query))
{
     var users_Id = reader.GetOrdinal("Users_Id");
}
```

I get an exception and the message is: "Read must be called first". How should I fix it?

**Comment by bgrainger** [comment]
This looks like an incompatibility between this connector and Connector/NET. (It's also inconsistent with `SqlDataReader` behaviour.)

The only workaround is to call `Read` (or `await ReadAsync`) before `GetOrdinal`, but that will read the first row so it might not be possible in your situation.

**Comment by bgrainger** [comment]
Fixed in [0.29.2](https://github.com/mysql-net/MySqlConnector/releases/tag/0.29.2).
